packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx

```diff
@@ -95,7 +95,7 @@ const ProofOfAddressContainer = ({
 
     const from_platform = getPlatformRedirect(app_routing_history);
 
-    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform.ref);
+    const should_show_redirect_btn = !isNavigationFromDerivGO() && Object.keys(platforms).includes(from_platform.ref);
```

Comment:
when `if (isNavigationFromDerivGO()) return { name: platform_name.DerivGO, route: '', ref: 'derivgo' };` is removed from the other file, wont `Object.keys(platforms).includes(from_platform.ref);` return false? so do we need to check for ` !isNavigationFromDerivGO()` condition again?

Reply:
Yeh u r right. Its not needed. I'll make the change